File: attachment-microservice/src/main/scala/data/Application.scala
package data

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.cloud.netflix.zuul.EnableZuulProxy

/**
  * Application
  *
  * @author 刘佳兴
  * @version V1.0
  */
object Application {

def main(args: Array[String]) {
    SpringApplication.run(classOf[Application])
  }

}

@SpringBootApplication
@EnableDiscoveryClient
@EnableZuulProxy
class Application

File: attachment-microservice/src/main/java/data/Application.java
package data;

import com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.web.WebMvcAutoConfiguration;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.cloud.netflix.zuul.EnableZuulProxy;
import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;

/**
 * @author 刘佳兴
 * @version V1.0
 */
@SpringBootApplication
@EnableDiscoveryClient
@EnableZuulProxy
public class Application extends WebMvcAutoConfiguration.WebMvcAutoConfigurationAdapter {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

    @Override
    public void configurePathMatch(PathMatchConfigurer configurer) {
        configurer.setUseSuffixPatternMatch(false);
        super.configurePathMatch(configurer);
    }

}
